test.sh

```diff
@@ -165,4 +247,13 @@ function runHTTPSTests() {
   ${conjur_test_cid} ${tests_command}
 }
 
+#function finish {
```

Comment:
this should be removed before merging

Reply:
We should have a cleanup for sure so this code should be there just not commented out for pull requests

Reply:
agreed, but I think it should be at the top of the file - that's our pattern elsewhere (though I'm blanking right now whether there's a technical reason - maybe it needs to be defined before main?)

ETA: ha, saw your comment below about moving the `trap` up

Reply:
yeah, put it in comments for troubleshooting purposes but will uncomment before final push

Reply:
moved up and uncommented